src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -245,7 +241,7 @@ string GetPathFromRegistry (string valueName)
 		protected bool BuildInternal (string projectOrSolution, string target, string [] parameters = null, Dictionary<string, string> environmentVariables = null)
 		{
 			buildLogFullPath = (!string.IsNullOrEmpty (BuildLogFile))
-				? Path.GetFullPath (Path.Combine (Root, Path.GetDirectoryName (projectOrSolution), BuildLogFile))
+				? Path.GetFullPath (Path.Combine (XABuildPaths.TopDirectory, Path.GetDirectoryName (projectOrSolution), BuildLogFile))
```

Comment:
Why shouldn't this keep using `Root`? And why should `Root` be changed?

Reply:
I didn't want to change `Root` as some parts use it as reference point in the `monodroid` repo and it does not equal the repository root